### PR TITLE
plugins/wdc: Fix code scanning alert

### PR DESCRIPTION
**Pull request by ikegami-t**
Fix the comparison of narrow type with wide type in loop condition.
  Note: Include some coding style errors fixes.

### CODE REVIEW

**Comment by ikegami-t** [comment]
Fix for the issue #2204.

File: plugins/wdc/wdc-nvme.c [hunk 1]
```diff
@@ -9648,10 +9648,10 @@ static int wdc_fetch_log_file_from_device(struct nvme_dev *dev, __u32 fileId,
 					  __u16 spiDestn, __u64 fileSize, __u8 *dataBuffer)
 {
 	int ret = WDC_STATUS_FAILURE;
-	__u32                     chunckSize = WDC_DE_VU_READ_BUFFER_STANDARD_OFFSET;
-	__u32                     maximumTransferLength = 0;
-	__u32                     buffSize = 0;
-	__u64                     offsetIdx = 0;
```
Comment (igaw) [inline plugins/wdc/wdc-nvme.c:9654]: Please do not mix whitespace changes with code changes.

Comment (ikegami-t) [reply plugins/wdc/wdc-nvme.c:9654]: Sorry for mixing the changes everytime so just split the changes to commits.

**Comment by igaw** [comment]
Please use a more fitting commit title.

**Comment by ikegami-t** [comment]
Sorry for too late to do this and just fixed the commit subject as below.
```diff
- plugins/wdc: Change for loop index variable type to __u32
+ plugins/wdc: Fix narrow index variable type in for loop to __u32
```

**Comment by igaw** [comment]
Thanks!

**Pull request merged**